openshift-hack/images/os/Dockerfile

```diff
@@ -1,26 +1,18 @@
+# fedora:29 is an image built within the release scripts:
+# https://github.com/openshift/release/blob/b45a09d248b8cdb8fe3bf5f3cfa0b4fee57d04c8/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.10.yaml#L63-L65
```

Comment:
That's wrong, that's only reflecting the mapping, we can change it here and we should change it. I can followup on that, and then we'll update release repo as well.